scss/login.scss

```diff
@@ -0,0 +1,66 @@
+.login-container {
+  margin: 100px auto;
+  width: fit-content;
+}
+
+.login-form {
+  background-color: $wild-sand;
+  border-top: solid 5px $cerulean;
```

Comment:
check [this](https://developer.mozilla.org/es/docs/Web/CSS/border-top#Valores) about the correct order of the properties values